seed/ts-sdk/accept-header/src/api/resources/service/client/Client.ts

```diff
@@ -60,8 +60,6 @@ export class Service {
                 "X-Fern-Runtime-Version": core.RUNTIME.version,
                 ...requestOptions?.headers,
             },
-            contentType: "application/json",
-            requestType: "json",
```

Comment:
@Swimburger this feels like a bug

Reply:
This was a bug that is no longer a bug. There is no request body in these endpoints, so the request content type and type should be omitted.